### DISCUSSION TITLE
docker simple setup

### DISCUSSION THREAD

**Discussion by oues81** [Kovah/LinkAce#740] [Support]
I'm trying to make this work for hours. I could get access to the browser interface landing page but still faced issues with the config and could not get past the database setup page, no matter what i try. I read everything i have found, that is, not much to help.

I'm using a lot of tool and framework but cant get past this simple install.

Is there someone who could help. Can get you the logs and infos needed.

Thanks by advance.

**Comment by Kovah** [comment]
Please post error messages or any other helpful information. What means "could not get past the database setup page"?
